javascript/javascript1/week3/areWeThereYet.js

```diff
@@ -0,0 +1,23 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 500,
+};
+
+/* Let's calculate total travel time in advance. I think, this will make codes easy to understand. */
+let totalTravelTime = ((travelInformation.speed)/60) * (travelInformation.destinationDistance);
+
+//console.log(travelInformation.speed);
+//console.log(travelInformation.destinationDistance); 
+//console.log(totalTravelTime);
+
+/*Considering that the same codes might be used for other travel time estimation, I am writing codes for more than and less than an hour long travel. */
+
+function toHoursAndMinutes() { if (totalTravelTime < 60) {

```

Comment:
Note: it makes sense to use make totalTravelTime to be argument of a function. 
Motivation: imagine you need to print time to travel for multiple travels in different places of your code (and even in different files)
It's not possible with current approach: variable that is visible inside of the function